application/controllers/validate_otp.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+require __DIR__ . '/../models/UserModel.php';
+require __DIR__ . '/destroy_session.php';
+
+session_start();
+
+if($_POST['otp'] == $_SESSION['otp']) {
+  $userTable = new UserModel('user');
+
+  if($_SESSION['task'] == 'register') {
+    if($userTable->addUser($_SESSION['email'], $_SESSION['password'], $_SESSION['fname'], $_SESSION['lname']) == TRUE) {
+      echo "<p class='alert'> You have been registered. </p>";
```

Comment:
Let's not use alert and instead display this message on UI

Reply:
Done.